### ISSUE TITLE
How to specify my own color theme?

### ISSUE THREAD

**Issue by koji-kojiro** [cxxxr/lem#56]
Hi,

I'd like to customize color theme of `lem`. So, I edited my `.lemrc` as follows.

```common-lisp
;; -*- mode:lisp -*-

(in-package :lem-user)

(define-color-theme "my-theme" ()
  (display-background-mode :light)
  (minibuffer-prompt-attribute :foreground "blue" :bold-p t)
  (region :foreground nil :background "#eedc82")
  (modeline :background "#bbbbbb" :foreground "black")
  (modeline-inactive :background "#bbbbbb" :foreground "#777777")
  (completion-attribute :foreground "#e5e5e5" :background "#0000FF")
  (non-focus-completion-attribute :foreground "black" :background "#aaaaaa")
  (syntax-string-attribute :foreground "#8B2252")
  (syntax-comment-attribute :foreground "#cd0000")
  (syntax-keyword-attribute :foreground "#C000A0")
  (syntax-constant-attribute :foreground "#ff00ff")
  (syntax-function-name-attribute :foreground "#0000ff")
  (syntax-variable-attribute :foreground "#8D5232")
  (syntax-type-attribute :foreground "#00875f"))

(load-theme "my-theme")

(setf (variable-value 'truncate-lines :global) nil)
(setf (variable-value 'lem.line-numbers:line-numbers :global) nil)

```

Here, "my-theme" is copied from `lem-core/color-theme.lisp` and just renamed.
However, this code crushes `lem`, with the error messages as follows.

```
odd number of &KEY arguments
Backtrace for: #<SB-THREAD:THREAD "editor" RUNNING {C2F0519}>
0: (LEM:SET-ATTRIBUTE LEM-USER::DISPLAY-BACKGROUND-MODE :LIGHT) [more]
...
```
As the conclusion, the error can be solved by commenting out the line `(display-background-mode :light)`.
In the present case, we cannot set default background or foreground with `define-color-theme` and we have to use `(setf (attribute-fg-color ...))` instead of that.
Is this expected behavior? or Bug?

Thanks.

**Comment by cxxxr** [comment]
Thank you for your report.
The cause is that `lem::display-background-mode` is not exported.
I fixed it with 1e186e1b8db10aa5ad70c55b2b03bb212312b06e

**Comment by koji-kojiro** [comment]
Thanks for quick feedback!
